front/app/modules/commercial/content_builder/admin/components/CraftComponents/Container/index.tsx

```diff
@@ -24,6 +25,7 @@ Container.craft = {
         (node) =>
           node.data.type === Text ||
           node.data.type === Image ||
+          node.data.type === Iframe ||
```

Comment:
Did you mention at some point that this rule was buggy or am I misremembering?

Reply:
Yeah, it was missing the "return" before, so it was buggy. This was fixed in the About Box PR though and is now in Master, so this is the working version :+1: